flixel/input/FlxKeyManager.hx

```diff
@@ -135,25 +141,37 @@ class FlxKeyManager<Key:Int, KeyList:FlxBaseKeyList> implements IFlxInputManager
 	 * @param	Status		The key state to check for.
 	 * @return	Whether the provided key has the specified status.
 	 */
-	public function checkStatus(KeyCode:Key, Status:FlxInputState):Bool
+	public inline function checkStatus(KeyCode:Key, Status:FlxInputState):Bool
```

Comment:
This function is too complex to be `inline`.

Reply:
I mostly did this to match `FlxGamepad.checkStatus` but it should work because checkStatus should always be called with a literal key value through inlined getters like `FlxG.keys.pressed.A` which is inlined and assumedly optimized all the way down the call stack to become:
```
	var key = getKey(KeyCode);
	if (key == null)
	{
		return false;
	}
	return key.hasState(Status);
```

otherwise only checkKeyArrayState should get the full inline. of course, people can call this method, as it's public and that can cause problems. I thought about making it private and adding `@:allow(flixel.input.FlxBaseKeyList)` to it. but that might screw up someone's project.